eng/Packages.Data.props

```diff
@@ -63,6 +63,7 @@
     <PackageReference Update="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="2.6.1" />
     <PackageReference Update="Microsoft.CodeAnalysis.FxCopAnalyzers" Version="2.6.2" />
     <PackageReference Update="Microsoft.CodeAnalysis" Version="2.3.0" />
+    <PackageReference Update="Microsoft.DotNet.ApiCompat" Version="5.0.0-beta.19552.1" />
```

Comment:
When I started looking into this problem, I noticed there's a few newer versions available. Should we upgrade while we're at it? That said, this is the version that's been downloaded, like, 68K times, while newer ones total to < 10. So safety in numbers?

Reply:
I was thinking to update the version but I didn't want to change too much at once. So after this all goes through we can try updating the version.